### PR TITLE
Revert "Small tomato killer tweak (#27265)"

### PR DESCRIPTION
**Pull request by metalgearsloth**
This reverts commit 4903bc2.

Resolves https://github.com/space-wizards/space-station-14/issues/27561

Apparently dozens to hundreds of these can be made in a round which the server is not made for. Botany's been a long-standing problem and it just manifested again.

:cl:
- remove: Revert killer tomatoes pending botany reworks due to performance issues.

### CODE REVIEW

**Comment by TheShuEd** [comment]
Maybe just remove prototypes, but dont touch imprinting mechanics?

File: Content.Shared/NPC/Components/FactionExceptionComponent.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ namespace Content.Shared.NPC.Components;
 /// Prevents an NPC from attacking ignored entities from enemy factions.
 /// Can be added to if pettable, see PettableFriendComponent.
 /// </summary>
-[RegisterComponent, NetworkedComponent, Access(typeof(NpcFactionSystem), typeof(SharedNPCImprintingOnSpawnBehaviourSystem))] // TO DO (Metalgearsloth): If we start adding a billion access overrides they should be going through a system as then there's no reason to have access, but I'll fix this when I rework npcs.
```
Comment (deltanedas) [inline Content.Shared/NPC/Components/FactionExceptionComponent.cs:10]: theres an api already @TheShuEd why didnt you use it noob

Comment (TheShuEd) [reply Content.Shared/NPC/Components/FactionExceptionComponent.cs:10]: ???

Comment (deltanedas) [reply Content.Shared/NPC/Components/FactionExceptionComponent.cs:10]: use api instead of bypassing it with access

Comment (TheShuEd) [reply Content.Shared/NPC/Components/FactionExceptionComponent.cs:10]: Idk how

Comment (deltanedas) [reply Content.Shared/NPC/Components/FactionExceptionComponent.cs:10]: public methods on NpcFactionSystem

**Comment by Emisse** [comment]
mald pr ban urself

**Pull request closed without merging**